stock_predictor/data_handler.py

```diff
@@ -0,0 +1,11 @@
+from qlib.contrib.data.handler import Alpha158
+
+class Alpha158TwoWeeks(Alpha158):
+    
+    """
+    Override the get_label_config method in Alpha158.
+    This will build a data handler that labels the items in different way.
+    Here we labels the item with the price increase within next 10 days.
```

Comment:
NIT: Here we label the items with price increase in next 10 trading days.

Reply:
Thanks for the advice. I have revised the sentence.
By the way, what's the meaning of NIT?

Reply:
https://en.wikipedia.org/wiki/Nitpicking#:~:text=Nitpicking%20is%20a%20term%2C%20first%20used%20in%201956%2C,lice%2C%20generally%20head%20lice%29%20from%20another%20person%27s%20hair.

Nitpicking is a term that describes the action of giving too much attention to unimportant detail.

I know its not important but still commented on it :)